googledatastore/pom.xml

```diff
@@ -47,4 +62,28 @@ LICENSE file.
       <scope>provided</scope>
     </dependency>
   </dependencies>
+  <build>
```

Comment:
why is this one showing up here in this PR? You might want to do a rebase again?